packages/calypso-e2e/src/jest-playwright-config/environment.ts

```diff
@@ -183,29 +210,59 @@ class JestEnvironmentPlaywright extends JestEnvironmentNode {
 	/**
 	 * Handle events emitted by jest-circus.
 	 */
-	async handleTestEvent( event: Circus.Event ) {
+	async handleTestEvent( event: Circus.Event, state: Circus.State ) {
```

Comment:
Calls Allure methods that correspond to the Jest emitted events for fine grained control over the reporting values.

The list of events are available [here](https://github.com/facebook/jest/blob/main/packages/jest-types/src/Circus.ts).